loggo/loggo.py

```diff
@@ -371,15 +371,11 @@ def _represent_return_value(self, response, truncate=140):
         """
         Make a string representation of whatever a method returns
         """
-        representable = (int, float, str, list, set, dict, type(None), bool, tuple)
-        if isinstance(response, representable):
-            return '({})'.format(self._force_string_and_truncate(response, truncate, use_repr=True))
         # some custom handling for request response objects
-        content = getattr(response, 'content', False)
-        if content:
-            return '({})'.format(self._force_string_and_truncate(content.decode('utf-8'), truncate, use_repr=True))
-        # fallback, should not happen
-        return ''
+        if str(type(response)) == "<class 'requests.models.Response'>":
```

Comment:
Using this instead of isinstance() so we don't need to import the requests library. Not sure at all if this is better than just making the import though.

Reply:
:|

Reply:
really there should be either zero or good handling of response objects imo

Reply:
You unhappy with this? The way I see it there's three options:

1. This.
2. Require and manage versions of the requests library in setup.py. Do the usual `isinstance` check here. I'd be happy with this as well, but it seemed a bit heavy for one type check.
3. No special handling for requests.Responses, which means they'll appear as something like '<Response [200]>' in the logs, and the response body will not be there.

Bitpanda needs the body to be there at the moment so I would go with option 1 or 2.

Reply:
yeah i guess I was never that satisfied with this special handling of `Response`. Nothing else gets special handling so it's a bit unpredictable for the user. If another object needs custom handling, there's no elegant way to add it in yet. But oh well, one step at a time I guess. So yeah this will do for now, would love if there was a note in the readme or somewhere that Responses get some nicer formatting though.

Reply:
Looks good to me. I agree, we should rethink custom handling if we add more exception but for now this should do it.